test/mocks/delinkedCalculation.js

```diff
@@ -0,0 +1,73 @@
+const mockDelinkedCalculation1 = {
+  applicationId: 1810909,
+  calculationId: 123456789,
+  sbi: 123456789,
+  frn: 1234567890,
+  paymentBand1: 30000,
+  paymentBand2: 50000,
+  paymentBand3: 150000,
+  paymentBand4: 9999999999.99,
+  percentageReduction1: 50,
+  percentageReduction2: 55,
+  percentageReduction3: 60,
+  percentageReduction4: 70,
+  progressiveReductions1: 15000,
+  progressiveReductions2: 11000,
+  progressiveReductions3: 65000,
+  progressiveReductions4: 104963.84,
+  referenceAmount: 195963.84,
+  progressiveReduction: 299948.34,
+  totalDelinkedPayment: 103984.50,
+  paymentAmountCalculated: 51992.25
```

Comment:
totalProgressiveReduction is missing from mocks too

Reply:
same as above

Reply:
renamed progressiveReduction to totalProgressiveReduction and added it in to the object